.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: continuous-integration
+on: [pull_request]
+jobs:
+  test-runner:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+        check-latest: true
+    -
```

Comment:
```suggestion
```
This is causing problems ig

Reply:
Thanks for the catch, @daemon1024 !!! Was doing too many things at once 😭